codex-rs/tui/src/slash_command.rs

```diff
@@ -44,3 +48,28 @@ impl SlashCommand {
 pub fn built_in_slash_commands() -> Vec<(&'static str, SlashCommand)> {
     SlashCommand::iter().map(|c| (c.command(), c)).collect()
 }
+
+#[cfg(test)]
```

Comment:
Again, I don't think this test derisks very much. It feels like it would only ever fail if we restructured how commands work, so it is unlikely to catch anything.

An integration test that verifies `/compact` would be a better way to verify this.